README.md

```diff
@@ -56,7 +55,7 @@ technologies - such as containers - to achieve isolation (see [usage](docs/usage
 
 We do not distribute Hermeto as a standalone package as of now.
 
-To install Hermeto for local development, see the [CONTRIBUTING.md](CONTRIBUTING.md).
+To install Hermeto for local development, see the [CONTRIBUTING.md](https://github.com/hermetoproject/hermeto/blob/main/CONTRIBUTING.md).
```

Comment:
I think with markdown it's fine as it is. See, unlike with GH pull request/issues IDs vs full URLs outside of the webUI context which are immutable (i.e. only in read-only mode via commit messages), these references in markdown are actually mutable, i.e. if we were to either rename the project again tomorrow or switch GH for GitLab, the link would have to be replaced again whereas the built-in linking/referencing mechanism in markdown prevents this. Even when reading the README file offline it mentions the CONTRIBUTING.md file directly which resides in the repo root directory, so I'm afraid we're not gaining anything with this hunk.

Reply:
Oh, I see it now. Sadly, 
https://github.com/mkdocs/mkdocs/discussions/3230
https://github.com/squidfunk/mkdocs-material/issues/6879
https://github.com/mkdocs/mkdocs/issues/608#issuecomment-109855759

none of the above solves the issue, sigh.
What you have is an acceptable trade-off, so let's keep it as is.